src/gui/systray.cpp

```diff
@@ -65,7 +65,8 @@ void Systray::setTrayEngine(QQmlApplicationEngine *trayEngine)
     _trayEngine->addImportPath("qrc:/qml/theme");
     _trayEngine->addImageProvider("avatars", new ImageProvider);
     _trayEngine->addImageProvider(QLatin1String("svgimage-custom-color"), new OCC::Ui::SvgImageProvider);
-    _trayEngine->addImageProvider(QLatin1String("unified-search-result-icon"), new UnifiedSearchResultImageProvider);
+    _trayEngine->addImageProvider(QLatin1String("unified-search-result-icon"), new TrayImageProvider);
+    _trayEngine->addImageProvider(QLatin1String("activity-item-icon"), new TrayImageProvider);
```

Comment:
why twice the same object used to create two different image providers ?
what is the benefit of duplication ?

Reply:
Missed this, going to unify them

Reply:
Done!